<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralWidget { 
/*background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(38, 147, 206, 255), stop:0.5 rgba(39, 149, 210, 255), stop:1 rgba(18, 109, 158, 255)); */
/* background-color: rgb(39,149,210) */
background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 #8DC26F, stop:1 #76b852);
}

QLabel { 
color: white; background: transparent;
}

QPushButton {
color: white;
background: #4CAF50;
border-radius: 0px;
}

QPushButton:hover {
background: #43A047
}

QPushButton:disabled {
background-color: rgb(204, 204, 204);
}</string>
   </property>
   <widget class="QLabel" name="lblHello">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>948</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>30</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Hello!</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblWelcome">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>948</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>Welcome to SRFRS! Login to get started.</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QStackedWidget" name="loginStacked">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>330</y>
      <width>411</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#loginStacked {
border:0;
background: white;
}


</string>
    </property>
    <widget class="QWidget" name="loginBox">
     <property name="styleSheet">
      <string notr="true">QLineEdit {
outline: 0;
background: #f2f2f2;
padding-left: 8px;
}

QLabel {
color: #b3b3b3;
font-size: 14px;
}

QPushButton#btnCreate {
background: 0;
color: #4CAF50;
font-size: 14px;
border-radius: 0px;
}</string>
     </property>
     <widget class="QLineEdit" name="txtUsername">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>331</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtPassword">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>110</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLogin">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>180</y>
        <width>38</width>
        <height>17</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>LOGIN</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lblCreate">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>250</y>
        <width>100</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Not registered?</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCreate">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>250</y>
        <width>112</width>
        <height>20</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Create an account</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="createBox">
     <property name="styleSheet">
      <string notr="true">QLineEdit {
outline: 0;
background: #f2f2f2;
padding-left: 8px;
}

QLabel {
color: #b3b3b3;
font-size: 14px;
}

QPushButton#btnSignin {
background: 0;
color: #4CAF50;
font-size: 14px;
border-radius: 0px;
}</string>
     </property>
     <widget class="QPushButton" name="btnRegister">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>210</y>
        <width>38</width>
        <height>17</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>REGISTER</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtRegisterUsername">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>140</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSignin">
      <property name="geometry">
       <rect>
        <x>311</x>
        <y>280</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Sign in</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtRegisterPassword">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>331</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblSignin">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>280</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Already registered?</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtUsername</tabstop>
  <tabstop>txtPassword</tabstop>
  <tabstop>btnLogin</tabstop>
  <tabstop>btnCreate</tabstop>
  <tabstop>txtRegisterUsername</tabstop>
  <tabstop>txtRegisterPassword</tabstop>
  <tabstop>btnRegister</tabstop>
  <tabstop>btnSignin</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
